app/views/step_by_step_pages/publish_or_delete.html.erb

```diff
@@ -28,14 +28,20 @@
     <% if @step_by_step_page.has_draft? %>
       <div class="govuk-panel panel-default">
         <div class="panel-body">
-          <p class="govuk-body">Before publishing check for broken links.</p>
-          <p class="govuk-body">After publishing add this step by step to a mainstream browse category and a taxonomy topic.</p>
-          <%= render "govuk_publishing_components/components/button", {
-            text: "Publish changes",
-            href: step_by_step_page_publish_path(@step_by_step_page)
-          } %>
+          <% if !@step_by_step_page.scheduled_for_publishing? %>
+            <p class="govuk-body">Before publishing check for broken links.</p>
+            <p class="govuk-body">After publishing add this step by step to a mainstream browse category and a taxonomy topic.</p>
+            <%= render "govuk_publishing_components/components/button", {
+              text: "Publish changes",
+              href: step_by_step_page_publish_path(@step_by_step_page)
+            } %>
+          <% end %>
           <% if user_can_schedule? %>
-            <p class="govuk-body govuk-!-margin-top-5">Or</p>
+            <!–– when the feature flag is removed, the conditional statement below should be romoved
+            and the 'Or' should be included in the conditional statement above ––>
+            <% if !@step_by_step_page.scheduled_for_publishing? %>
```

Comment:
I'd move this to line 39, i.e. `<% if user_can_schedule? && !@step_by_step_page.scheduled_for_publishing? %>`

Reply:
This wouldn't work because there is an if-else statement below, which determines whether Schedule or Unschedule button is shown, so the 'Unschedule publishing' button would never appear. 
I couldn't think of a better way of doing this without messing indentation.